src/plugins/tts/plugin.tts.js

```diff
@@ -251,7 +251,9 @@ BookReader.prototype.ttsPlayPause = function() {
     this.ttsToggle();
   } else {
     this.ttsEngine.togglePlayPause();
-    this.ttsUpdateState(this.ttsEngine.paused);
+    // extra ttsUpdateState call?
+    // already bound to play/pause button in initNavbar
+    //this.ttsUpdateState(this.ttsEngine.paused);
```

Comment:
I noticed ttsUpdateState being called twice when clicking play/pause button. If this is deemed unnecessary I will remove the call and comments entirely.

Reply:
We might be able to remove this ; but it probably make more sense to remove from initNavBar. 
Although it being called twice is ok it just syncs the state, so it might need to happen twice at various points in the pipeline to ensure the state is in sync.

Reply:
I'm ok with keeping it then. I can add it back and take out the param to prevent confusion if that works